lib/logflare_web/controllers/api/source_controller.ex

```diff
@@ -35,7 +36,7 @@ defmodule LogflareWeb.Api.SourceController do
 
   def show(%{assigns: %{user: user}} = conn, %{"token" => token}) do
     with source when not is_nil(source) <- Sources.get_by(token: token, user_id: user.id),
-         [source] <- Sources.preload_for_dashboard([source]) do
+         source <- Sources.preload_defaults(source) do
```

Comment:
It always match, do we need it to be part of `with`?

Reply:
doesn't need to be, in this case it is more of a stylistic choice since we can't pipe in the `do`.

Reply:
switched it to use `=`